docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -1,204 +1,239 @@
+   displayed, then click **Kubernetes**.
+
+1. Copy the command to add the `teleport-agent` chart repository and paste 
+   it in a terminal on a workstation where `kubectl` is installed.
+
+1. Type the Teleport service namespace and the display name to use when connecting to 
+   this cluster namespace, then click **Next**.
+
+   After you click Next, Teleport generates a script to configure and enroll the 
+   Kubernetes cluster as a resource in the Teleport cluster.
+
+1. Copy the command displayed in the Teleport Web UI and run it in a terminal with access
+   to your Kubernetes cluster.
+
+   The Teleport Web UI displays "Successfully detected your new Kubernetes cluster" as 
+   confirmation that your cluster is enrolled. When you see this message, click **Next**
+   to continue.
+
+## Step 3/4. Test Kubernetes access
+
+You can now set up access for specific Kubernetes groups and users to test access to
+the Kubernetes cluster you just enrolled.
+
+To set up and test access:
+
+1. Type a Kubernetes group name and, optionally, one or more Kubernetes user names that should 
+   have access to Kubernetes resources, then click **Next**.
+
+   You must specify at least one Kubernetes group. If you don't specify a Kubernetes user, you can
+   connect to the cluster using your Teleport user by default.
+
+1. (Optional) Specify the namespace, a Kubernetes group from the previous step, and 
+   either your Teleport user or a Kubernetes user, then click **Test Connection**.
+   
+   ![Connect to the Kubernetes cluster](../../img/k8s/test-k8s-connection.png)
+
+1. (Optional) Copy and run the commands displayed in the Teleport Web UI to interact with the
+   Kubernetes cluster to verify access through Teleport.
+   
+   ```code
+   $ tsh login --proxy=<Var name="teleport.example.com"/>:443 --auth=local --user=<Var name="admin@example.com"/> <Var name="teleport.example.com"/>
+   $ tsh kube login <Var name="Kubernetes-cluster-name"/>
+   $ tsh kubectl get pods
+   ```
+
+1. Click **Finish**.
+
+1. Click **Browse Existing Resources** to see your Kubernetes cluster and discovered 
+   applications.
+
+## Step 4/4. Configure roles and authorize access
+
+To authenticate to a Kubernetes cluster using Teleport, you must have a
+Teleport role that grants access to the cluster you plan to interact with
+through at least one Kubernetes user or group.
+
+The following example illustrates how to configure a `kube-access` role for 
+one Kubernetes group named `viewers` and one Kubernetes user.
+
+1. Create a file called `kube-access.yaml` with the following content:
+   
+   ```yaml
+   kind: role
+   metadata:
+     name: kube-access
+   version: v7
+   spec:
+     allow:
+       kubernetes_labels:
+         '*': '*'
+       kubernetes_resources:
+         - kind: '*'
+           namespace: '*'
+           name: '*'
+           verbs: ['*']
+       kubernetes_groups:
+       - viewers
+       kubernetes_users:
+       - <Var name="myuser"/>
```

Comment:
This one is implicit if you don't have any kubernetes_users defined on all roles.
If you have multiple users defined it will fail so we shouldn't encourage people using it otherwise it will fail.

Reply:
Unfortunately, the page already includes this change on `master` from the PR that this backports. I'll open a separate PR!